tests/streaming_test.py

```diff
@@ -523,6 +523,30 @@ async def test_account_activity_subs_success(self, ws_connect):
             }
         })
 
+    @no_duplicates
```

Comment:
Based on our convo, here's a list of the things I recommend you test: 

1. Edge case (described below): Unsubbing from stream to which you're not currently subbed. Research into how TDA actually handles this is necessary. 
2. Unsub failure, because that *might* be a thing that happens? Copy the "return status 21" tests below might be appropriate here. Ignore this if this is actually how TDA handles case 1. 
3. Standard behavior: sub then unsub. For realism, let's actually test "sub, receive one piece of data, unsub." 
4. Vague, but very powerful: a test that fails without the lock.